ark/jsonschema/string.ts

```diff
@@ -0,0 +1,26 @@
+import { rootSchema, type Intersection } from "@ark/schema"
+import type { Out, Type } from "arktype"
+import { JsonSchemaScope, type StringSchema } from "./scope.ts"
+
+export const parseStringJsonSchema: Type<
+	(In: StringSchema) => Out<Type<string, any>>,
+	any
+> = JsonSchemaScope.StringSchema.pipe((jsonSchema): Type<string> => {
+	const arktypeStringSchema: Intersection.Schema<string> = {
+		domain: "string"
+	}
+
+	if ("maxLength" in jsonSchema)
+		arktypeStringSchema.maxLength = jsonSchema.maxLength
+	if ("minLength" in jsonSchema)
+		arktypeStringSchema.minLength = jsonSchema.minLength
+	if ("pattern" in jsonSchema) {
+		if (jsonSchema.pattern instanceof RegExp) {
+			arktypeStringSchema.pattern = [
```

Comment:
`@ark/schema` can parse a RegExp instance or a string, so easier to just let it handle it directly here


```suggestion
		if (jsonSchema.pattern instanceof RegExp)
			arktypeStringSchema.pattern = jsonSchema.pattern
```

Reply:
Done! 94cf1fb8